src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -179,6 +179,11 @@
         no-access? (= accessLevel "NO ACCESS")
         auth-domain (:authorizationDomain workspace)
         domain-groups (map :membersGroupName auth-domain)]
+    ;; GAWB-2872
+    (let [namespace (:namespace workspace-id)
+          name (:name workspace-id)]
+      (assert (some? namespace) (str "Cannot list workspace because it has no namespace." (if (some? name) (str " Name is \"" name "\"."))))
+      (assert (some? name) (str "Cannot list workspace because it has no name." (if (some? namespace) (str " Namespace is \"" namespace "\".")))))
```

Comment:
`some?` for testing not-null is basically a no-op, you can just `(assert namespace ...)` and `(if name ...)`. Also, these `if`s should be `when`s since they lack a negative case.